packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -393,9 +393,15 @@ export class GutenbergEditorPage {
 		}
 
 		const navbarComponent = new NavbarComponent( this.page );
-		const actions: Promise< unknown >[] = [
-			this.page.waitForNavigation( { url: '**/home/**', waitUntil: 'load' } ),
-		];
+
+		// There are three different places you can return to, depending on how you entered the editor.
+		const navigationPromise = Promise.race( [
+			this.page.waitForNavigation( { url: '**/home/**' } ),
+			this.page.waitForNavigation( { url: '**/posts/**' } ),
+			this.page.waitForNavigation( { url: '**/pages/**' } ),
+		] );
```

Comment:
Oh weird!

I just tested in non-proxied prod, and still saw this behavior. Specifically, I went to the `Posts` page, clicked "Add new post", then returned, and it dropped me back in `**/posts/**`

Reply:
What was the workflow you were doing were it would differ and take you back to home?

Reply:
Interesting! Maybe this is due to my lack of cross-verification - I've always started new posts from the Home dashboard, and it looks like the editor's back function remembers where you came from and adjusts the text accordingly.

Given this new (to me) knowledge, I wonder if it would make sense to rename the method from `returnToHomeDashboard` to something else more representative of the actual function - `returnToCalypso`?

Reply:
Great idea! Updated! 👍